tools/coq_dune.ml

```diff
@@ -0,0 +1,262 @@
+(************************************************************************)
+(*         *   The Coq Proof Assistant / The Coq Development Team       *)
+(*  v      *   INRIA, CNRS and contributors - Copyright 1999-2018       *)
+(* <O___,, *       (see CREDITS file for the list of authors)           *)
+(*   \VV/  **************************************************************)
+(*    //   *    This file is distributed under the terms of the         *)
+(*         *     GNU Lesser General Public License Version 2.1          *)
+(*         *     (see LICENSE file for the text of the license)         *)
```

Comment:
This file should be dual-licensed MIT / LGPL 2.1 if the intent is to move it to Dune at some point (this will make things easier if there were contributions from other people in the meantime).

Reply:
Good point thanks, anyways the file may be too simple to claim copyright for it.

Reply:
Updated.